x-pack/solutions/search/plugins/search_homepage/public/components/header/stateful_promo.tsx

```diff
@@ -14,20 +14,21 @@ export const StatefulHeaderPromo = () => {
   return (
     <HeaderPromo
       title={i18n.translate('xpack.searchHomepage.header.statefulPromo.title', {
-        defaultMessage: 'Build your first search solution',
+        defaultMessage: 'GPUs go brrr! GPU-accelerated inference for ELSER',
       })}
       description={i18n.translate('xpack.searchHomepage.header.statefulPromo.description', {
         defaultMessage:
-          'Learn the fundamentals of creating a complete search experience with this hands-on tutorial.',
+          'We are thrilled to launch ELSER on ElS -- get state-of-the-art semantic search relevance without having to manage your own machine learning nodes.',
       })}
       cta={
         <HeaderCTALink
           data-test-subj="searchHomepageSearchHomepageHeaderCTA"
-          data-telemetry-id="8-search-tutorial"
-          href="https://www.elastic.co/search-labs/tutorials/search-tutorial/welcome"
+          // "search-promo-homepage-" is prepended to the telemetry id in HeaderCTALink
+          data-telemetry-id="10-elser-on-eis"
+          href="https://www.elastic.co/docs/explore-analyze/elastic-inference/eis#elser-on-eis"
         >
           {i18n.translate('xpack.searchHomepage.statefulPromo.content', {
-            defaultMessage: 'Start the tutorial',
+            defaultMessage: 'Learn more',
```

Comment:
Can this be something other than 'Learn more' => 'View the documentation' is at least better as a clearer marker for where the link will take me.
cc/ @lcawl